UnitTestBot/UTBotJava
# Move issue to In Progress status when a branch is linked to it

## Description

Automate changing status of an issue to "In Progress" when a branch/PR is linked to it.

## How to test

### Manual tests

1. Select an issue that is in "To Do" status
2. Create a branch
3. Add a commit to it with test "Fixes #<issue number>"
4. Open a PR
5. Check that status of the issue is automatically changed to "In Progress"

## Self-check list
